eventkit_cloud/jobs/models.py

```diff
@@ -385,6 +389,29 @@ class Meta:  # pragma: no cover
     def __str__(self):
         return '{0}'.format(self.name)
 
+
+class UserJobActivity(models.Model):
+    CREATED = 'created'
+    VIEWED = 'viewed'
+    UPDATED = 'updated'
+    DELETED = 'deleted'
+
+    user = models.ForeignKey(User)
+    job = models.ForeignKey(Job)
+    type = models.CharField(max_length=100, blank=False)
+    created_at = models.DateTimeField(default=timezone.now, editable=False)
+
+
+def user_owns_job(user=None, job_uid=None):
+    if not job_uid or not user:
+        return False
+    job = Job.objects.get(uid=job_uid)
+    if job.user == user or job.published:
```

Comment:
Job.user now represents the job creator not the "owner".  The owner would be any person with admin permissions.